src/dpdkstat.c

```diff
@@ -447,26 +459,26 @@ static int dpdk_helper_run (void)
     struct timespec ts;
     cdtime_t now = cdtime();
     cdtime_t half_sec = MS_TO_CDTIME_T(1500);
```

Comment:
Can you give this variable a more meaningful name, e.g. `safety_period`? Alternatively, please add a comment explaining what this does. Also, is it on purpose that the "half second" variable contains, in fact, one and a half seconds?